DMTN-306.tex

```diff
@@ -1,79 +1,119 @@
-% Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
-% See LPM-51 for version number policy.
-\setDocChangeRecord{%
-  \addtohist{1}{YYYY-MM-DD}{Unreleased.}{Fabio Hernandez}
-}
+This paper is structured as follows. In section \ref{section-data-movement-use-cases} we present the main data movement use cases we need to satisfy and section \ref{section-data-movement-tools} presents what tools were selected or developed and how they are composed to implement solutions to those use cases.
 
+\begin{figure}[h]
+\includegraphics[width=\textwidth]{images/RubinDataFacilities.png}
+\caption{Images flow from the Summit Site, where the telescope is located in Chile, to the Base Site and then to the three Rubin Data Facilities which collectively provide the computational capacity for processing the images taken by the Observatory for the duration of the survey.}
+\label{fig:data-facilities}
+\end{figure}
 
-\begin{document}
+\section{Data movement use cases}
+\label{section-data-movement-use-cases}
 
-% Create the title page.
-\maketitle
-% Frequently for a technote we do not want a title page  uncomment this to remove the title page and changelog.
-% use \mkshorttitle to remove the extra pages
+This section presents three distinct use cases for moving data among the data facilities used by the Rubin Observatory.
+
+\subsection{From summit to archive}
+\label{summit-to-archive}
+
+The data acquisition system stores each exposure as a set of approximately 200 files, one per sensor on the camera focal plane. Once an exposure is recorded at the summit site, its constituent files are transferred in parallel to an object store at the archive center via the S3 protocol. To optimize these transfers over the international network linking the summit to the SLAC archive site, we employ specialized network connection pooling, keep-alive mechanisms, and TCP tuning.
+
+Given that raw images undergo prompt processing for transient object detection and alert generation, the target end-to-end latency for transferring a single exposure—including data compression and other overheads—is set to seven seconds for four gigabytes of compressed data.
+
+Ancillary data (e.g., telemetry, specialized databases) are replicated to the archive center using native protocols to ensure timeliness. Additionally, a small number of certified calibration files are transferred infrequently from the archive to the summit and other locations.
+
+\subsection{From archive to processing facilities and back}
+\label{summit-to-data-facilities}
+
+Annual processing of the entire image dataset recorded since the beginning of the survey is carried out across three facilities: the US Data Facility, hosted at SLAC National Laboratory in California, USA\footnote{https://www.slac.stanford.edu}, the French Data Facility, hosted by the IN2P3 computing center (CC-IN2P3) in Lyon, France\footnote{https://cc.in2p3.fr}, and the UK Data Facility, operated by the LSST:UK consortium\footnote{https://www.lsst.ac.uk}.
```

Comment:
```suggestion
Annual processing of the entire image dataset recorded since the beginning of the survey is carried out across three facilities: the US Data Facility, hosted at SLAC National Accelerator Laboratory in California, USA\footnote{https://www.slac.stanford.edu}, the French Data Facility, hosted by the IN2P3 computing center (CC-IN2P3) in Lyon, France\footnote{https://cc.in2p3.fr}, and the UK Data Facility, operated by the LSST:UK consortium\footnote{https://www.lsst.ac.uk}.
```

Reply:
Implemented